analytics/terraform/spark-eks-ipv6/helm-values/yunikorn-values.yaml

```diff
@@ -0,0 +1,148 @@
+
+image:
+  repository: apache/yunikorn
+  tag: scheduler-${image_version}
+  pullPolicy: Always
+
+pluginImage:
+  repository: apache/yunikorn
+  tag: scheduler-plugin-${image_version}
+  pullPolicy: Always
+
+nodeSelector: {}
+tolerations: []
+affinity: {}
+
+configuration: null    # deprecated; use queues.yaml in yunikornDefaults
+operatorPlugins: null  # deprecated; use service.operatorPlugins in yunikornDefaults
+placeHolderImage: null # deprecated; use service.placeholderImage in yunikornDefaults
+
+admissionController:
+  image:
+    repository: apache/yunikorn
+    tag: admission-${image_version}
+    pullPolicy: Always
+  replicaCount: 1
+  serviceAccount: yunikorn-admission-controller
+  hostNetwork: true
+  resources:
+    requests:
+      cpu: 200m
+      memory: 500Mi
+    limits:
+      cpu: 500m
+      memory: 500Mi
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  service:
+    type: ClusterIP
+  processNamespaces: null # deprecated; use admissionController.processNamespaces in yunikornDefaults
+  bypassNamespaces: null  # deprecated; use admissionController.bypassNamespaces in yunikornDefaults
+  labelNamespaces: null   # deprecated; use admissionController.labelNamespaces in yunikornDefaults
+  noLabelNamespaces: null # deprecated; use admissionController.noLabelNamespaces in yunikornDefaults
+
+web:
+  image:
+    repository: apache/yunikorn
+    tag: web-${image_version}
+    pullPolicy: Always
+  resources:
+    requests:
+      memory: 500Mi
+      cpu: 500m
+    limits:
+      memory: 500Mi
+      cpu: 500m
+
+service:
+  type: ClusterIP
+  port: 9080
+  portWeb: 9889
+
+ingress:
+  enabled: false
+  ingressClassName: ""
+  annotations: {}
+  hosts:
+    - host: chart-example.local
+      paths: []
+  pathType: Prefix
+  tls: []
+
+resources:
+  requests:
+    cpu: 400m
+    memory: 2Gi
+  limits:
+    cpu: 4
+    memory: 2Gi
+
+
+
+# When this flag is true, the admission controller will be installed along with the scheduler.
+# When this flag is false, the admission controller will not be installed.
+# Once the admission controller is installed, all traffic will be routing to yunikorn.
+embedAdmissionController: true
+
+# When this flag is true, the scheduler will be deployed as Kubernetes scheduler plugin.
+# When this flag is false, the scheduler will be deployed as a standalone scheduler.
+enableSchedulerPlugin: false
+
+
+# Bootstrap configuration for YuniKorn - will be rendered into yunikorn-defaults ConfigMap.
+# Any valid options for YuniKorn may be specified here.
+# Use this link for more values -> https://yunikorn.apache.org/docs/user_guide/service_config/#yunikorn-configuration
+yunikornDefaults:
+  # The default volume bind timeout value of 10 seconds may be too short for EBS.
+  service.volumeBindTimeout: "60s"
+  service.placeholderImage: registry.k8s.io/pause:3.7
+  service.operatorPlugins: "general,spark-k8s-operator"
+  admissionController.filtering.bypassNamespaces: "^kube-system$"
```

Comment:
I *think* this reg-ex is matching the job spark-operator is spawning when it's uninstalling. 
I had the cleanup.sh script get stuck waiting for that job, which in turn was stuck pending for some reason. 

trying again with the namespace matching inverted looks to have cleared that problem but I'm not SURE why. 
`admissionController.filtering.processNamespaces: "^spark-team"`

Reply:
The cluster was deleted clean for me.